### DISCUSSION TITLE
What does "interlaced output" actually do

### DISCUSSION THREAD

**Discussion by syrz05** [ntsc-rs/ntsc-rs#130] [Q&A]
Sorry if this is a stupid question but I enabled interlaced output and the video is still not interlaced, so I'm wondering if its supposed to somehow interlace videos and there's just more I have to do to get it working, or if it does something else entirely.

**Comment by smfthe1199** [comment]
Hey! I have to oblige, I checked on the "interlaced output" thing and the video **DID** get interlaced; there are lines in the head switching thing and when deinterlacing is enabled, **there is a difference**.

**Reply by syrz05** [reply]
https://github.com/valadaptive/ntsc-rs/assets/174294465/a755574d-6bb7-4aed-be38-e8f987fb98e9

Here's a test video I made, there is no visible interlacing, I also rendered a version with interlaced output disabled and there is no difference between the two.

**Reply by smfthe1199** [reply]
The thing is that you'll notice the lines down here in the head switching thing, and not the contents of the video itself.

**Reply by syrz05** [reply]
So it doesn't do this? 
![image](https://github.com/valadaptive/ntsc-rs/assets/174294465/5a54a006-e9be-4c51-ac01-ad21f6775c05)

**Reply by smfthe1199** [reply]
Yep!

**Reply by Degamisu** [reply]
> So it doesn't do this? ![image](https://private-user-images.githubusercontent.com/174294465/344752475-5a54a006-e9be-4c51-ac01-ad21f6775c05.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk5MjAxMzUsIm5iZiI6MTcxOTkxOTgzNSwicGF0aCI6Ii8xNzQyOTQ0NjUvMzQ0NzUyNDc1LTVhNTRhMDA2LWU5YmUtNGM1MS1hYzAxLWFkMjFmNjc3NWMwNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwMlQxMTMwMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wOTRlNzljZjNhNGQwMDZhMTFmNDY1ZTQzNzNkZTg3N2QyYjI2YzkzM2VkMDg4NmU4NzM1MzEzODFiNzhiZjM1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3UOnZdKwkewhiBGHS1RlelvjB2rPdMDwMcd1qLWB1o4)

No. The video must be interlaced _first_ to get the weaving artifacts. Progressive video (non interlaced) will just be downconverted, and deinterlaced. VLC Media Player has a "Phosphor" deinterlacer, and if it is set correctly, will give you the weaving artifacts you want!

[VLC Media Player (if you dont already have it)](https://www.videolan.org/vlc/)
[Phosphor Filter Documentation](https://wiki.videolan.org/Deinterlacing/#Phosphor_.28v1.2.0.2B.29)

**Reply by smfthe1199** [reply]
I was referring to this.

**Reply by walkerpatricia706-spec** [reply]
I do interlacing with alight motion using stripes. Just set Count to 20.00, then select punch out and Screen space

**Comment by valadaptive** [comment]
The "Interlaced output" checkbox determines whether the video metadata says the video is interlaced. Many media players will automatically *de-interlace* such media upon playback. If you're using VLC, you can try disabling it via the Video > Deinterlace menu.

**Reply by smfthe1199** [reply]
I tried upon opening a video it doesn't interlace and it doesn't interlace the entire video, **the weaving artifacts** gets interlaced.

**Reply by dawnch0rus** [reply]
the input video has to be interlaced itself

**Reply by smfthe1199** [reply]
Here's a better explained video I made of what's going on:

https://drive.google.com/file/d/17ZlB8YLgmjpKsmdmSVPB-BaffsuwFUEH/view?usp=sharing

I made a grammar mistake, I am supposed to write "look at it" not "look in it" -_- , forgot to correct that.
